product-service/handler.mjs

```diff
@@ -107,4 +112,69 @@ export const createProduct = async(event) => {
       body: JSON.stringify(`There was an error executing createProduct - ${err}`),
     };
   }
+}
+
+export const catalogBatchProcess = async(event) => {
+  const products = event.Records.map(({body}) => body)
+  const ddbClient = new DynamoDBClient({ region: REGION });
+  const batchProducts = []
+
+  const params = {
+    TableName: process.env.PRODUCT_TABLE_NAME,
+  };
+
+  const paramsStock = {
+    TableName: process.env.STOCK_TABLE_NAME,
+  }
+
+  do {
+    const product = products.shift()
+    const item = product.split(',')
+    const productId = nanoid();
+    
+    params.Item = {
+      id: {S: productId},
+      title: { S: item[0]},
+      description: { S: item[1]} ,
+      price: { N: item[2]}
+    }
```

Comment:
you can use `marshall` and `unmarshall` functions from `@aws-sdk/util-dynamodb` to deal with serialisation of dynamodb entities and avoid manual wrapping

Reply:
@sergeyklykov  Oh, I didn't know about that, I'll take a look. Thank you!